### DISCUSSION TITLE
[Bug] Unable to get the logs from my application through Opentelemetery Collector configuration to Apache Skywalking

### DISCUSSION THREAD

**Discussion by lijie123bes** [apache/skywalking#12715] [Q&A]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/skywalking/issues?q=is%3Aissue) and found no similar issues.

### Apache SkyWalking Component

OAP server (apache/skywalking)

### What happened

This is my Otelcol config file

receivers: otlp: protocols: grpc:

processors: batch: exporters: otlp: endpoint: "localhost:11800" tls: insecure: true logging: loglevel: debug

service: pipelines: metrics: receivers: [otlp] processors: [batch] exporters: [logging,otlp]

This is my skywalking configuration file

receiver-sharing-server: selector: ${SW_RECEIVER_SHARING_SERVER:default} default:
While running the opentelemetry collector using otel configuration file below is the command that i am using

C:\otelcol.exe --config C:\opentelemetry-collector\otelcol\otelcol_configuration.yaml

Getting this error 2023-12-04T20:44:40.603+0530 warn zapgrpc/zapgrpc.go:195 [core] [Channel #1 SubChannel #2] grpc: addrConn.createTransport failed to connect to {Addr: "localhost:11800", ServerName: "localhost:11800", }. Err: connection error: desc = "transport: Error while dialing: dial tcp [::1]:11800: connectex: No connection could be made because the target machine actively refused it." {"grpc_log": true}

### What you expected to happen

read logs from my application

### How to reproduce

# For HTTP server
restHost: ${SW_RECEIVER_SHARING_REST_HOST:0.0.0.0}
restPort: ${SW_RECEIVER_SHARING_REST_PORT:0}
restContextPath: ${SW_RECEIVER_SHARING_REST_CONTEXT_PATH:/}
restMaxThreads: ${SW_RECEIVER_SHARING_REST_MAX_THREADS:200}
restIdleTimeOut: ${SW_RECEIVER_SHARING_REST_IDLE_TIMEOUT:30000}
restAcceptQueueSize: ${SW_RECEIVER_SHARING_REST_QUEUE_SIZE:0}
httpMaxRequestHeaderSize: ${SW_RECEIVER_SHARING_HTTP_MAX_REQUEST_HEADER_SIZE:8192}

# For gRPC server
gRPCHost: ${SW_RECEIVER_GRPC_HOST:0.0.0.0}
gRPCPort: ${SW_RECEIVER_GRPC_PORT:0}
maxConcurrentCallsPerConnection: ${SW_RECEIVER_GRPC_MAX_CONCURRENT_CALL:0}
maxMessageSize: ${SW_RECEIVER_GRPC_MAX_MESSAGE_SIZE:0}
gRPCThreadPoolQueueSize: ${SW_RECEIVER_GRPC_POOL_QUEUE_SIZE:0}
gRPCThreadPoolSize: ${SW_RECEIVER_GRPC_THREAD_POOL_SIZE:0}
gRPCSslEnabled: ${SW_RECEIVER_GRPC_SSL_ENABLED:false}
gRPCSslKeyPath: ${SW_RECEIVER_GRPC_SSL_KEY_PATH:""}
gRPCSslCertChainPath: ${SW_RECEIVER_GRPC_SSL_CERT_CHAIN_PATH:""}
gRPCSslTrustedCAsPath: ${SW_RECEIVER_GRPC_SSL_TRUSTED_CAS_PATH:""}
authentication: ${SW_AUTHENTICATION:""}

### Are you willing to submit a pull request to fix on your own?

- [X] Yes I am willing to submit a pull request on my own!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by wu-sheng** [comment]
> No connection could be made because the target machine actively refused it.

This looks like firewall or something.
